Lib/fontTools/otlLib/builder.py

```diff
@@ -2574,7 +2574,8 @@ def add(self, glyphs):
             return
         self.classes_.add(glyphs)
         for glyph in glyphs:
-            assert glyph not in self.glyphs_
+            if glyph in self.glyphs_:
+                raise ValueError(f"Glyph {glyph} is already present in class.")
```

Comment:
there's an `OpenTypeLibError` custom exception in `fontTools.otlLib.error` module, you should use that instead

Reply:
Done. However, I'm unsure if there's anything useful I can pass in for the location?

Reply:
set location to None, it will not be printed